Add differentiation between commands from different Python versions (#6514)
wazuh/wazuh-documentation: source/cloud-security/amazon/services/prerequisites/dependencies.rst

It is recommended to use a pip version greater than or equal to 19.3 to ease the installation of the required dependencies.

.. tabs::

>    .. group-tab:: Python 3.7 - 3.10

Reviewer: ```suggestion
   .. group-tab:: Python 3.7–3.10
```

      .. code-block:: console

         # pip3 install --upgrade pip

   .. group-tab:: Python 3.11

      .. code-block:: console

         # pip3 install --upgrade pip --break-system-packages
   
      .. note::
         

>          The ``--break-system-packages`` parameter is required to make the install on the default externally managed environment (more information on the `PEP 668 description <https://peps.python.org/pep-0668/>`_). To avoid it, the command can be executed inside a virtual environment but this would require a modification on the ``aws-s3`` script shebang to use the environment's interpreter. 

Reviewer: ```suggestion
         This command modifies the default externally managed Python environment. See the `PEP 668 <https://peps.python.org/pep-0668/>`__ description for more information.
         
         To prevent the modification, you can run ``pip3 install --upgrade pip`` within a virtual environment. You must update the ``aws-s3`` script shebang with your virtual environment interpreter, for example, ``#!/path/to/your/virtual/environment/python``.
```

To install the dependencies, execute the following command:

.. tabs::

>    .. group-tab:: Python 3.7 - 3.10

Reviewer: ```suggestion
   .. group-tab:: Python 3.7–3.10
```

      .. code-block:: console

         # pip3 install boto3==1.17.85 botocore==1.20.85 jmespath==0.9.5 python-dateutil==2.8.1 six==1.14.0 urllib3==1.26.5 s3transfer==0.4.2 pyarrow==8.0.0 numpy==1.21.6
   
   .. group-tab:: Python 3.11

      .. code-block:: console

         # pip3 install --break-system-packages boto3==1.17.85 botocore==1.20.85 jmespath==0.9.5 python-dateutil==2.8.1 six==1.14.0 urllib3==1.26.5 s3transfer==0.4.2 pyarrow==8.0.0 numpy==1.21.6

      .. note::
         

>          In the case of using a virtual environment, the ``--break-system-packages`` parameter should be removed from the command.

Reviewer: ```suggestion
         If you're using a virtual environment, remove the ``--break-system-packages`` parameter from the command above.
```
